base/process.jl

```diff
@@ -72,6 +72,13 @@ backticks, e.g.
 """
 Cmd
 
+hash(x::Cmd) = hash(x.exec, hash(x.env, hash(x.ignorestatus, hash(x.dir, hash(x.flags)))))
+function ==(x::Cmd, y::Cmd)
+    a = isequal(x.exec, y.exec) && isequal(x.env, y.env)
```

Comment:
I know, but I'm not a huge fan of doing that for stylistic reasons. Happy to switch it to that if either that's the preferred style for the project or there are some (supposedly minor but worth having over a simple issue like this) benefits to not setting these values to local variables and having them as a one-liner.